src/main/php/plans.php

```diff
@@ -20,54 +20,44 @@
 </head>
 <body>
 <section class="section-plans">
-<?php include("universal-header.php");  ?>
-    <div class="plans-header">
-        <h1>Learn about our plans here!</h1>
-        <br
+    <?php include("universal-header.php");  ?>
+
+    <h1>Learn about our plans here!</h1>
+
+    <div class="plans-block">
+        <img src="https://images.theconversation.com/files/339674/original/file-20200604-130929-iutdtd.jpg?ixlib=rb-1.1.0&rect=8%2C0%2C5455%2C3637&q=45&auto=format&w=496&fit=clip" alt="Bronze">
+        <h3>Bronze</h3>
+        <ul>
+            <li>Our bronze plan is the cheapest!</li>
+            <li>Unlimited access to the gym</li>
+        </ul>
     </div>
-    <div>
-        <div class="images">
-            <div class="Width">
-                <img src="https://images.theconversation.com/files/339674/original/file-20200604-130929-iutdtd.jpg?ixlib=rb-1.1.0&rect=8%2C0%2C5455%2C3637&q=45&auto=format&w=496&fit=clip" alt="Bronze" class="plans-image-1">
-            <p>Bronze</p>
-                <div class="plans-description">
-                    <ul>
-                        <li>Our bronze plan is the cheapest!</li>
-                        <li>Unlimited access to the gym</li>
-                        <br>
-                    </ul>
-                </div>
-            </div>
-            <div class="Width">
-                <img src="https://static01.nyt.com/images/2020/01/06/well/04run-gym/04run-gym-superJumbo.jpg" alt="Silver" class="plans-image-2" >
-            <p>Sliver</p>
```

Comment:
Small typo!

Reply:
this code was removed hence the red :).
additions are below.

Reply:
this code was removed hence the red :).
additions are below.